src/Backend/SgateShopgatePlugin/Models/Export/Product.php

```diff
@@ -814,8 +885,9 @@ public function prepareBasePrice($price, $detail)
 
             $basePrice .= " / {$referenceUnit} {$unit}";
         } elseif ($this->getPurchaseSteps($detail) > 1) {
+            $label = $detail->getUnit()->getName();
```

Comment:
Looks like `$detail->getUnit()` could be returning null in this branch of the `if` statement. Shouldn't we keep the fall back to "pro Stück" here?